source/features/comments-time-machine-links.tsx

```diff
@@ -105,7 +105,7 @@ async function showTimemachineBar(): Promise<void | false> {
 	}
 
 	addNotice(
-		<>You can also <a className="rgh-link-date" href={String(url)}>view this object as it appeared at the time of the comment</a> (<relative-time datetime={date}/>)</>,
+		<>You can also <a className="rgh-link-date" href={String(url)} data-pjax="#repo-content-pjax-container">view this object as it appeared at the time of the comment</a> (<relative-time datetime={date}/>)</>,
```

Comment:
```suggestion
		<>You can also 
			<a 
				className="rgh-link-date"
				href={String(url)}
				data-pjax="#repo-content-pjax-container"
			>
			view this object as it appeared at the time of the comment
			</a> (<relative-time datetime={date}/>)</>,
```

Too many things to be on one line. (Don't commit this its going to fail linting)

Reply:
I was wondering why linter doesn't give warnings on these super long lines

Reply:
We don't have an issue with the long line so much. The issue is having so many attributes on the same line. It gets hard to read.

Reply:
What I do sometimes is extract the JSX out so it's a single variable. e.g. 
```jsx
const link = (
	<a className="rgh-link-date" href={String(url)} data-pjax="#repo-content-pjax-container">
		view this object as it appeared at the time of the comment
	</a>
);
```
```suggestion
		<>You can also {link} (<relative-time datetime={date}/>)</>,
```